lib/rspec/templates/spec_file_template.erb

```diff
@@ -1,7 +1,8 @@
 require "spec_helper"
 
-describe "<%= @raml.title %>" do
-  include Rack::Test::Methods<% if @examples.generate! && @examples.compose.size > 0 %>
+RSpec.describe "<%= @raml.title %>", type: :request do
+<% if @examples.generate! && @examples.compose.size > 0 %>
 
```

Comment:
So I can add `<%-` to the ERB setting so that is will not create a newline. I didn't care because the tests were already broken.